exercises/bowling/bowling.spec.js

```diff
@@ -65,7 +65,7 @@ describe('Bowling', () => {
       expect(bowling.score()).toEqual(81);
     });
 
-    xtest('a strike in the last frame gets a two roll bonues that is counted once', () => {
+    xtest('a strike in the last frame gets a two roll bonuses that is counted once', () => {
```

Comment:
It should be `bonus` as in the test case above and below. Also ref: https://github.com/exercism/problem-specifications/blob/master/exercises/bowling/canonical-data.json#L87